microsoft-support.md

```diff
@@ -0,0 +1,37 @@
+# Microsoft support for grpc-dotnet
+
+[Microsoft supports grpc-dotnet](https://LINK-TODO) on multiple operating systems, per the [Microsoft Modern Lifecycle](https://support.microsoft.com/help/30881/modern-lifecycle-policy).
```

Comment:
The linked page would be something like this: https://dotnet.microsoft.com/platform/support/policy/aspnet

It hasn't been written yet so I don't know the exact content.